.maintain/reset-spec/peregrine_stg_relay.py

```diff
@@ -92,80 +103,119 @@ def update_spec(input: typing.Dict):
                 "para_assignment": "5FxM6yYBFvMeX3C2QrQDXJ8LhhzG5sPjgUtX77cX1wihHhVq",
                 "authority_discovery": "5FxM6yYBFvMeX3C2QrQDXJ8LhhzG5sPjgUtX77cX1wihHhVq"
             }
+        ],
+        [
+            acc_eve,
+            acc_eve,
+            {
+                "grandpa": "5GnPNFKvLRy9FF8N1G9YjGmjJA4cUsUC7WgEx3rDeMFnZsXk",
+                "babe": "5GKaEkaA8NVdpsruRcnpeLBNGzMcFsEfwEY3Jq7Vmw9brztR",
+                "im_online": "5GKaEkaA8NVdpsruRcnpeLBNGzMcFsEfwEY3Jq7Vmw9brztR",
+                "para_validator": "5GKaEkaA8NVdpsruRcnpeLBNGzMcFsEfwEY3Jq7Vmw9brztR",
+                "para_assignment": "5GKaEkaA8NVdpsruRcnpeLBNGzMcFsEfwEY3Jq7Vmw9brztR",
+                "authority_discovery": "5GKaEkaA8NVdpsruRcnpeLBNGzMcFsEfwEY3Jq7Vmw9brztR"
+            }
+        ],
+        [
+            acc_ferdie,
+            acc_ferdie,
+            {
+                "grandpa": "5CPW6uFwdjoHTj14C1VWiK96Cj2sJBALbC964zHaGAni3J2S",
+                "babe": "5CSYQMyi7iGVuHLgLNDXcpPXZgvWWrP7mqd1sHdBUSeafXf5",
+                "im_online": "5CSYQMyi7iGVuHLgLNDXcpPXZgvWWrP7mqd1sHdBUSeafXf5",
+                "para_validator": "5CSYQMyi7iGVuHLgLNDXcpPXZgvWWrP7mqd1sHdBUSeafXf5",
+                "para_assignment": "5CSYQMyi7iGVuHLgLNDXcpPXZgvWWrP7mqd1sHdBUSeafXf5",
+                "authority_discovery": "5CSYQMyi7iGVuHLgLNDXcpPXZgvWWrP7mqd1sHdBUSeafXf5"
+            }
         ]
     ]
     input["genesis"]["runtime"]["sudo"]["key"] = acc_alice
     input["genesis"]["runtime"]["staking"].update({
-        "validatorCount": 4,
+        "validatorCount": 6,
         "stakers": [
             [
                 acc_alice,
                 acc_alice,
-                1000000000000,
+                1000000000000000,
                 "Validator"
             ],
             [
                 acc_bob,
                 acc_bob,
-                1000000000000,
+                1000000000000000,
                 "Validator"
             ],
             [
                 acc_charlie,
                 acc_charlie,
-                1000000000000,
+                1000000000000000,
                 "Validator"
             ],
             [
                 acc_dave,
                 acc_dave,
-                1000000000000,
+                1000000000000000,
+                "Validator"
+            ],
+            [
+                acc_eve,
+                acc_eve,
+                1000000000000000,
+                "Validator"
+            ],
+            [
+                acc_ferdie,
+                acc_ferdie,
+                1000000000000000,
                 "Validator"
             ]
         ]
     })
     input["genesis"]["runtime"]["configuration"]["config"].update(
         {
             "max_code_size": 3145728,
-            "max_head_data_size": 20480,
```

Comment:
default values where taken expect for `validation_upgrade_delay` and `validation_upgrade_cooldown`